src/PerfView/StackViewer/CallTreeView.cs

```diff
@@ -314,6 +322,13 @@ public void ValidateTree()
 #endif
         }
 
+        public void SetBackgroundColor(BackgroundColor color)
+        {
+            State = ((int)color).ToString();
+        }
+
+        public string State { get; set; }
```

Comment:
Why is this not a field called BackgroundColor?   I suspect it was when you though this was the model, but this a view object so it is very reasonable for it to have display things in it.  The type should also be color.